configs/autopush_shared.ini

```diff
@@ -0,0 +1,43 @@
+; Settings shared by the connection and endpoint nodes.
+
+; Enable verbose logging for WebSocket connections and HTTP requests.
+; This should be disabled in production.
+debug = false
```

Comment:
autopush: error: debug is a flag but is being set to 'false'

Getting that with `debug = false`.